index.html

```diff
@@ -1,10 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<title>Document</title>
-<link rel="stylesheet" href="/assets/style.css" /></head>
-<body>
-	<h1>Hi</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="https://use.typekit.net/yri5dks.css" />
+    <title>Document</title>
+
+    
+  <link rel="stylesheet" href="/assets/style.css" /></head>
+  <body>
+    <div class="start-screen-block">
+      <header>
+        <ul>
+          <li><a href="#">HOME</a></li>
+          <li><a href="#">PORTFOLIO</a></li>
+          <li><a href="#">BLOG</a></li>
+          <li><a href="#">PAGES</a></li>
+          <li><a href="#">ABOUT US</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+      </header>
+      <div class="header-logo">
+        <img src="src/img/logo.png" style="width:50px;height:50px" />
```

Comment:
Say "NO" to inline-styles. It's related to the whole document.

Reply:
Done